BRLS/src/library/fixitem/FixItemUI.java

```diff
@@ -4,99 +4,82 @@
 
 public class FixItemUI {
 
-	private enum uI_sTaTe { INITIALISED, READY, INSPECTING, COMPLETED };
-
-	private fIX_iTeM_cONTROL CoNtRoL;
-	private Scanner InPuT;
-	private uI_sTaTe StAtE;
+    private enum FixItemUIState {INITIALISED, READY, INSPECTING, COMPLETED };
 
+    private FixItemControl control;
+    private Scanner scanner;
+    private FixItemUIState uiState;
 	
-	public FixItemUI(fIX_iTeM_cONTROL CoNtRoL) {
-		this.CoNtRoL = CoNtRoL;
-		InPuT = new Scanner(System.in);
-		StAtE = uI_sTaTe.INITIALISED;
-		CoNtRoL.SeT_Ui(this);
-	}
-
+    public FixItemUI(FixItemControl fixItemControl) {
+        this.control = fixItemControl;
+        scanner = new Scanner(System.in);
+        uiState = FixItemUIState.INITIALISED;
+        fixItemControl.setUI(this);
+    }
 
-	public void RuN() {
-		DiSpLaY_OuTpUt("Fix Item Use Case UI\n");
+    public void run() {
+        displayOutput("Fix Item Use Case UI\n");
 		
-		while (true) {
+        while (true) {
 			
-			switch (StAtE) {
+            switch (uiState) {
```

Comment:
Defect 1 PR#2.
Indentation needs to be addressed for the nested switch statements as per rule 9 of the coding style guideline.

Reply:
I changed it, can you check

Reply:
fixed, please review again

Reply:
Looks good.